### PR TITLE
Add tutorials

### PR DESCRIPTION
**Pull request by diegoamc**
Extract the tutorials page from prezento and add them here. In the process the tutorials were reviewed and new sections were included.

Also, add custom images for each tutorial.

### CODE REVIEW

**Comment by rafamanzo** [comment]
Would be nice to have the reference to the images source and to know their licenses.

File: _posts/2016-04-04-tutorial-keywords.markdown [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+The interpretation given to a _metric result_ (e.g., "Good", "Complex", "A"). Readings combined with ranges provide interpretations for _metric results_.
+
+### Reading Group
+A set of _readings_ that make sense when grouped together (e.g., "Good" and "Bad", "Simple" and "Complex").
+
+### Configuration
+A set of _metrics_.
+
+### Metric Result
+It is a numerical result associated to a metric by the Collector or by aggregation in the case of a Compound Metric.
```
Comment (danielkza) [inline _posts/2016-04-04-tutorial-keywords.markdown]: How do hotspot metrics/metric results fit into this description?

**Comment by danielkza** [comment]
- Was it intentional to replace the header image for all pages? Even the home shows the "Don't give up" picture instead of the Mezuro logo.
- The white text over the article images on the home is a bit hard to read. It might be a good idea to add some text outline or shadow.
- The GSoC idea posts look a bit out of context on the home page. We should prefix their titles with "Improvement Idea" or something similar to improve that.

File: _includes/head.html [hunk 2]
```diff
@@ -10,7 +10,7 @@
   <link rel="alternate" type="application/rss+xml" title="{{ site.title }}" href="{{ "/feed.xml" | prepend: site.baseurl | prepend: site.url }}" />
 
   <!-- Material Design Lite css Library -->
-  <link rel="stylesheet" type="text/css" href="https://storage.googleapis.com/code.getmdl.io/1.0.0/material.indigo-pink.min.css">
+  <link rel="stylesheet" type="text/css" href="https://storage.googleapis.com/code.getmdl.io/1.1.3/material.indigo-pink.min.css">
```
Comment (rafamanzo) [inline _includes/head.html]: Do we want to modify the [source template](https://github.com/gdg-managua/jekyll-mdl/)?

**Comment by rafamanzo** [comment]
Nice work with the `post.html` and `posts-cards` refactor.

**Comment by rafamanzo** [comment]
The current version breaks menu behaviour:
- [x] Opening it does not dim the body (a black shadow on the content)
- [x] It is not possible to close it again

It may be related to the MDL css version update.

**Comment by rafamanzo** [comment]
Nice job.

**Pull request merged**